Light.json

```diff
@@ -58810,6 +58837,64 @@
         "lore": "Blank"
       },
       "legacy": false
+    },
+    {
+      "id": 6848,
+      "side": "Light",
+      "rarity": "U1",
+      "set": "Virtual Premium Set",
+      "front": {
+        "title": "•Red 12",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/VirtualPremium-Light/large/red12.gif?raw=true",
+        "type": "Starship",
+        "subType": "Starfighter: X-Wing",
+        "uniqueness": "*",
+        "destiny": "2",
+        "power": "3",
+        "maneuver": "4",
+        "hyperspeed": "5",
+        "deploy": "2",
+        "forfeit": "4",
+        "icons": [
+          "Nav Computer"
+        ],
+        "characteristics": [
+          "Rebel (starship)"
+        ],
+        "gametext": "May add 1 pilot. If Puck Naeco piloting, he may be placed in Used Pile instead of Lost Pile when he is forfeited. Immune to attrition < 4.",
+        "lore": "Blank."
+      },
+      "pulledBy": [
+        "All Wings Report In & Darklighter Spin"
+      ],
+      "matching": [
+        "Puck"
+      ],
+      "legacy": false
+    },
+    {
+      "id": 6849,
+      "side": "Light",
+      "rarity": "C2",
+      "set": "Virtual Premium Set",
+      "front": {
+        "title": "•Puck",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/VirtualPremium-Light/large/puck.gif?raw=true",
+        "type": "Character",
+        "subType": "Rebel",
+        "destiny": null,
```

Comment:
Should not have null, but just leave destiny out.  Alternatively, we can now start updating these types of special values with the appropriate strings if we want: "7 or 1".  This isn't a blocker to merge, my script will remove the null with my next check-in so don't worry too much about it.

Reply:
This PR was created before string properties, so I had to follow null precedent.  Easily changeable as you said, but would like to do that in a future scenario beyond this PR.